src/task-2-2.js

```diff
@@ -0,0 +1,78 @@
+var { join, extname, basename } = require('path');
+var { createWriteStream } = require('fs');
+var { readdir, mkdir } = require('fs');
+
+var csv = require('csvtojson');
+
+// check csv folder on .csv files and work with them if they exists
+function csvManager() {
+  readdir(join(__dirname, '..', 'csv'), function (err, files) {
+    if (!err) {
+      files.forEach(file => {
+        var isCSV = extname(file) === '.csv';
+        if (isCSV) {
+          txtConverter(file);
+        }
+      });
+    } else {
+      console.log(err);
+    }
+  });
+}
+// provide .txt files with required information into txt folder
+function txtConverter(file) {
```

Comment:
naming conventions

Reply:
can you at least say what's wrong or suggest something?)